server/game/cards/06-WoE/CatchAndRelease.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('underscore');
+const Card = require('../../Card.js');
+
+class CatchAndRelease extends Card {
+    // Play: Return each creature to its owner`s hand. Each player
+    // discards random cards from their hand until they have 6 or
+    // fewer cards in hand. Gain 2 chains.
+    setupCardAbilities(ability) {
+        this.play({
+            effect: "return all creatures to their owner's hand",
+            gameAction: [
+                ability.actions.returnToHand((context) => ({
+                    target: context.game.creaturesInPlay
+                })),
+                ability.actions.gainChains({ amount: 2 })
+            ],
+            then: {
+                alwaysTriggers: true,
+                gameAction: ability.actions.sequential([
```

Comment:
I believe this one is over complicated.

Can you do simply: 
```
ability.actions.discard((context) => ({
   target: context.player.hand.length > 6 ? _.shuffle(context.player.hand).slice(0,6)
                        }))
```

?

No need for conditional or resolve ability

Reply:
Ah smart, thanks.  Fixed!